validation/standard/standard-config-params-sepolia-dev-0.toml

```diff
@@ -1,4 +1,14 @@
 # Standard Chain Config Parameters
+[rollup_config]
+# plasma must be nil
+seq_window_size = [3600, 3600]
+block_time = [2, 2]
+
+[optimism_portal_2]
+proof_maturity_delay_seconds = [1200, 604800]       # 7 days
+dispute_game_finality_delay_seconds = [600, 302400] # 2.5 days
```

Comment:
maybe adjust comments

Reply:
This file has now been removed due to changes on the base branch which I merged to this one [03b5bc2](https://github.com/ethereum-optimism/superchain-registry/pull/383/commits/03b5bc2f71946f702eb76a675044f11e4bf2c5cd).